#!/usr/bin/env python
PACKAGE = "gate_detect"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("cannyLow",                 int_t,    	0, "Low threshold of Canny Filter",	                        50,            1,  200)
gen.add("houghLinesThreshold",      int_t,    	0, "Low threshold of Hough Filter",	                        50,            1,  400)
gen.add("houghLinesMinLength",      int_t,    	0, "Min Line Length for Hough Filter",	                    50,            1,  400)
gen.add("houghLinesMaxLineGap",     int_t,    	0, "Max Line Gap for Hough Filter",	                        50,            1,  400)
gen.add("poleMax",                  int_t,    	0, "Max distance between two lines to represent a pole",	50,            1,  400)


exit(gen.generate(PACKAGE, "gateDetect", "gatedetect"))

